demos/asr-oms-monitoring/azuredeploy.parameters.us.json

```diff
@@ -13,6 +13,9 @@
         },
         "location": {
             "value": "usgovvirginia"
+        },
+        "psModulesUri": {
+            "value": "https://devopsgallerystorage.blob.core.windows.net/packages/"
```

Comment:
if this is required, then set this as the defaultValue in the template and remove this from here

Reply:
After making the required modifications, the Best Practices violation tag has been added with an  error log: Found hardcoded reference to core.windows.net Line: 46, Column: 60

Reply:
If we set this as the default value in the template, the TTK throws best practice violation error for hard coded url. Should we do it still?

Reply:
it's a bug in the TTK